src/utils/math/transformNumeric.spec.ts

```diff
@@ -11,3 +11,7 @@ test('Returns string values as is', () => {
 test('Returns "undefined" when no value is given', () => {
   expect(transformNumeric()).toEqual('')
 })
+
+test('Suffixes numeric values with "px" string, if value is zero the result should be 0px', () => {
```

Comment:
Could you please rename this test suit to `Outputs CSS for 0 as a value`.

Reply:
renamed test suite